scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -0,0 +1,728 @@
+#!/bin/bash
+
+# Defaults
+MIRRORING=false
+ADD_IMAGE_PULL_SECRET=false
+GA_RELEASE=true
+KERNEL_CONFIG_MC_FILE="./96-kata-kernel-config-mc.yaml"
+SKIP_NFD="${SKIP_NFD:-false}"
+TRUSTEE_URL="${TRUSTEE_URL:-"http://kbs-service:8080"}"
+CMD_TIMEOUT="${CMD_TIMEOUT:-900}"
+
+# Function to check if a command is available
+function check_command() {
+    local cmd="$1"
+    if ! command -v "$cmd" &>/dev/null; then
+        echo "$cmd command not found. Please install the $cmd CLI tool."
+        return 1
+    fi
+}
+
+# Function to wait for the operator deployment object to be ready
+function wait_for_deployment() {
+    local deployment=$1
+    local namespace=$2
+    local timeout=$CMD_TIMEOUT
+    local interval=5
+    local elapsed=0
+    local ready=0
+
+    while [ $elapsed -lt "$timeout" ]; do
+        ready=$(oc get deployment -n "$namespace" "$deployment" -o jsonpath='{.status.readyReplicas}')
+        if [ "$ready" == "1" ]; then
+            echo "Operator $deployment is ready"
+            return 0
+        fi
+        sleep $interval
+        elapsed=$((elapsed + interval))
+    done
+    echo "Operator $deployment is not ready after $timeout seconds"
+    return 1
+}
+
+# Function to wait for service endpoints IP to be available
+# Example json for service endpoints. IP is available in the "addresses" field
+#  "subsets": [
+#    {
+#        "addresses": [
+#            {
+#                "ip": "10.135.0.25",
+#                "nodeName": "coco-worker-1.testocp.local",
+#                "targetRef": {
+#                    "kind": "Pod",
+#                    "name": "controller-manager-87ffb6bfd-5zzvf",
+#                    "namespace": "openshift-sandboxed-containers-operator",
+#                    "uid": "00059394-29fb-44bf-8121-d1df02524ea8"
+#                }
+#            }
+#        ],
+#        "ports": [
+#            {
+#                "port": 443,
+#                "protocol": "TCP"
+#            }
+#        ]
+#    }
+#]
+
+function wait_for_service_ep_ip() {
+    local service=$1
+    local namespace=$2
+    local timeout=$CMD_TIMEOUT
+    local interval=5
+    local elapsed=0
+    local ip=0
+
+    while [ $elapsed -lt "$timeout" ]; do
+        ip=$(oc get endpoints -n "$namespace" "$service" -o jsonpath='{.subsets[0].addresses[0].ip}')
+        if [ -n "$ip" ]; then
+            echo "Service $service IP is available"
+            return 0
+        fi
+        sleep $interval
+        elapsed=$((elapsed + interval))
+    done
+    echo "Service $service IP ($ip) is not available after $timeout seconds"
```

Comment:
I think the `($ip)` should go to the success path.

Reply:
yeah my bad.. Sorry about it